### PR TITLE
PBI-14-15: update delete comment

### LINKED ISSUE #31: PBI-14: Teacher & Creator Can delete comment of announcement in class
**Story Point: 5**

### LINKED ISSUE #32: PBI-15: Creator Can edit comment of announcement in class
**Story Point: 13**

### PR DESCRIPTION
**Pull request by kanziebub**

### CODE REVIEW

**Review by AldiNFitrah — APPROVED** [review]
That's very neat and clean, thanks!

File: src/Controllers/CommentController.php [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+            $comment->content = $request->input('content');
+            $comment->save();
+
+            return ApiResponse::success($comment->toArray());
+        } catch (Exception $e) {
+            return ApiResponse::failed($e);
+        }
+    }
+
+    public function delete($id)
```
Comment (IlmaMannix) [inline src/Controllers/CommentController.php:83]: I'm sorry if my question is such a stupid question, but If I'm a teacher, could I delete my student comment?

Comment (kanziebub) [reply src/Controllers/CommentController.php:83]: That's not a stupid question la <3
I think there was a misunderstanding on my part, I thought editing and deleting could only be done by the creator of the comment.
Fixing it rn, thanks for the reminder ^^

**Review by nofamex — APPROVED** [review]
LGTM

**Review by IlmaMannix — APPROVED** [review]
LGTM, thankss for your hard workk <3

**Pull request merged**